src/components/filters/AttributeFilter/tests/AttributeDropdown.spec.tsx

```diff
@@ -63,9 +63,7 @@ describe("AttributeDropdown", () => {
         const attributeDisplayForm = createADF();
         const wrapper = renderComponent({ attributeDisplayForm });
         wrapper.find("button.s-country").simulate("click");
-        expect(
-            document.querySelectorAll(".gd-attribute-filter-overlay .s-attribute-filter-list-loading"),
-        ).toHaveLength(1);
+        expect(document.querySelectorAll(".s-isLoading.loading-mask")).toHaveLength(1);
```

Comment:
you shouldn't use classes that do not start with s- in tests (.loading-mask).
Why do we need `document.querySelectorAll`?

Reply:
I guess `document.querySelectorAll` is used instead of `wrapper.find` because the overlay is not inside the attribute filter DOM. But I don't really understand all those tests, I only modified them to pass, no refactoring done on my side.